src/test/java/org/jongo/AggregateTest.java

```diff
@@ -129,35 +140,35 @@ public void canAggregateWithMultipleDocuments() throws Exception {
     }
 
     @Test
-    public void canAggregateParameters() throws Exception {
+    public void canAggregateParameters() {
 
-        Iterator<Article> articles = collection.aggregate("{$match:{tags:#}}", "pandemic").as(Article.class);
+        ResultsIterator<Article> articles = collection.aggregate("{$match:{tags:#}}", "pandemic").as(Article.class);
```

Comment:
I suppose we can use here the new `articles` class variable

Reply:
Yes, it is a missing.